cartridges/openshift-origin-cartridge-jenkins-client/bin/jenkins_build

```diff
@@ -45,62 +113,113 @@ def is_cancelled
   queueItem = jobs_info["queueItem"]
 end
 
+# Schedules a new build.
 def schedule_build
-  status_code = `curl -s -w %{http_code} --output /dev/null -X POST --insecure https://${JENKINS_USERNAME}:${JENKINS_PASSWORD}@#{@hostname}#{@job_url}/build`
-  unless ['302', '200', '201'].include? status_code
-    puts "ERROR - Couldn't schedule job"
-    exit 292
```

Comment:
I verified this exit code is _not_ on the broker's categorized list, so dropping it should not be an issue.